src/components/directional-link-box/index.tsx

```diff
@@ -30,6 +31,9 @@ function DirectionalLinkBox({
 
 	return (
 		<Link
+			// Next.js's Link will attempt to refocus itself after a page transition, in order to keep the user focused on what it believes to be the same content. (https://github.com/vercel/next.js/issues/49386) This makes sense for a menu buttons/links, but not for content buttons/links. Setting unique ids & keys convince's Next that this is now a new element
+			id={id || undefined}
+			key={id || undefined}
```

Comment:
I think it makes more sense to have the comment outside of the component.

I don't think the `id || undefined` is needed because `id` either has a value or it will be undefined. WDYT?
```suggestion
  // Next.js's Link component will maintain a focus state even after a page transition (with static regeneration) (https://github.com/vercel/next.js/issues/49386). Adding a key attribute breaks that behaviour.
	return (
		<Link
			id={id}
			key={id}
```

Reply:
good point.